### PR TITLE
Add Alpine Linux support

### PR DESCRIPTION
**Pull request by roedie**
Since I use Alpine I added support for it. Not sure if you want to merge, but here it is.

### CODE REVIEW

File: templates/defaults.cfg [hunk 1]
```diff
@@ -49,7 +49,7 @@ defaults
     option {{ option }}
 {% endfor -%}
 {% endif -%}
-{% if ansible_distribution != 'CentOS' %}
+{% if ansible_distribution != 'CentOS' and ansible_distribution != 'Alpine' %}
```
Comment (benwebber) [inline templates/defaults.cfg]: Why block this out for Alpine?

Comment (roedie) [inline templates/defaults.cfg]: Alpine does not ship the error files. So, I figured they should not be included in the default config for Alpine.

Comment (benwebber) [inline templates/defaults.cfg]: I see, thanks. As-is this means users can't actually override `haproxy_defaults` and set error files if they install the files out-of-band.

We need to provide the user with a mechanism for configuring the files if they want to on these distributions. That's out of scope for this PR, but I'll log an issue for it.

Comment (roedie) [inline templates/defaults.cfg]: I might have a stab at this if I have some time.

**Pull request merged**